mx.py

```diff
@@ -13654,16 +13654,7 @@ def help_(args):
             abort('mx: command \'{0}\' is ambiguous\n    {1}'.format(name, ' '.join(hits)))
 
     command = _mx_commands.commands()[name]
-    doc = command.command_function.__doc__
-    if command.has_documentation():
-        format_args = []
-        if command.usage_msg:
-            format_args.append(command.usage_msg)
-        if command.doc_function:
-            format_args.append(command.doc_function())
-
-        doc = doc.format(*format_args)
-    print 'mx {0} {1}\n\n{2}\n'.format(name, command.usage_msg, doc)
+    print '\nmx {0} {1}\n\n{2}\n{3}\n'.format(name, command.usage_msg, command.command_function.__doc__, command.get_doc())
```

Comment:
If there's no documentation i think it would be better if it just prints the usage line.
Maybe only if there's nothing (no usage, no docstring, no doc function) should it print that there is no documentation.

I'm also not sure we need the leading `\n`

Reply:
@gilles-duboscq I thought the leading `\n` looked nicer, since the help message wouldn't be squeezed up against the terminal prompt (if that makes sense).